pkg/kube/pod_command_executor.go

```diff
@@ -21,6 +21,20 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+type ExecError struct {
```

Comment:
I suggest you add this:

```go
func (e *ExecError) Unwrap() error {
    return e.error
}
```

This will allow calling errors.Unwrap(err) for *ExecError that will return the underlying error. It might be useful later.

P.S. See more: https://pkg.go.dev/errors#pkg-overview